### DISCUSSION TITLE
想请教下怎么导出文章为markdown格式的

### DISCUSSION THREAD

**Discussion by carolin-violet** [halo-dev/halo#7138] [Q&A]
### Prerequisites

- [x] I have searched for related issues in the [Issues](https://github.com/halo-dev/halo/issues) list.
- [x] This is a feature related to Halo. If it is not an issue with the project itself, it is recommended to submit it in the [Discussions](https://github.com/halo-dev/halo/discussions).
- [x] If it is a feature suggestion for plugins and themes, please submit it in the respective plugin and theme repositories.

### Your current Halo version

2

### Describe this feature

当前halo导出文章导出的是html，手动改成markdown后看了下格式不太对，想请问下有什么方法能导出正常的markdown吗，就是和github上的README.md一样的格式那种

**Comment by ruibaby** [comment]
https://www.halo.run/store/apps/app-SUvBR

**Reply by carolin-violet** [reply]
我的halo是2.15的不支持呢，怎么升级到2.2呀大佬

**Reply by ruibaby** [reply]
每一种部署方式后面应该都有升级方式，https://docs.halo.run/category/%e5%ae%89%e8%a3%85%e6%8c%87%e5%8d%97/
